### PR TITLE
Changed ShowcaseView

### PR DESCRIPTION
**Pull request by Ananthusubramanian**
Solution to issue #1506 
Changed the showcaseview so that it closed on touch outside.
Fix: Now the showcaseview closes on touch outside
Changes:Changed the showcaseview

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java [hunk 1]
```diff
@@ -55,7 +55,8 @@ protected void onCreate(Bundle savedInstanceState) {
         navigationView.setOnNavigationItemSelectedListener(this);
 
          nav_home = (BottomNavigationItemView) findViewById(R.id.navigation_home);
-         nav_cam = (BottomNavigationItemView) findViewById(R.id.navigation_camera);
+
+        nav_cam = (BottomNavigationItemView) findViewById(R.id.navigation_camera);
```
Comment (sridharjajoo) [inline app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: Kindly remove this extra line 58

Comment (Ananthusubramanian) [reply app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: Removed the extra line.

**Comment by Ananthusubramanian** [comment]
I changed the extra line .

**Comment by codedsun** [comment]
@Ananthusubramanian Kindly squash the commits.

**Comment by anantprsd5** [comment]
Please squash your commits.

**Comment by Ananthusubramanian** [comment]
Squashed all the commits

**Comment by Ananthusubramanian** [comment]
@codedsun @anantprsd5 @sridharjajoo  I did the squashing of all the commits .

**Comment by codedsun** [comment]
@Ananthusubramanian Kindly update the branch.

**Comment by Ananthusubramanian** [comment]
@codedsun Updated the branch

**Comment by Ananthusubramanian** [comment]
@codedsun @anantprsd5 @sridharjajoo I squashed all the commits and update my branch.

**Comment by Ananthusubramanian** [comment]
@codedsun @anantprsd5 @sridharjajoo  I have squashed all my commits and updated my branch .

**Pull request merged**